flake.nix

```diff
@@ -97,7 +102,24 @@
         default = with pkgs; stdenv.mkDerivation {
           pname = "dune";
           version = "n/a";
-          src = ./.;
+          src =
+            let fs = lib.fileset; in
+            fs.toSource {
+              root = ./.;
+              fileset = fs.unions [
+                ./bin
+                ./boot
+                ./configure
+                ./dune-project
+                ./dune-file
+                ./src
+                ./plugin
+                ./vendor
+                ./otherlibs
+                ./Makefile
```

Comment:
is this the set of files where changes can trigger recompilation? If so, I think the makefile should be absent from this list. We should not use it as part of the build process. It is only used for development.

Reply:
Ithe Makefile is used for the nix build since the derivation doesn’t have a buildPhase, so it needs to be present